tests/functional/test_list.py

```diff
@@ -58,6 +174,57 @@ def test_uptodate_flag(script, data):
         'list', '-f', data.find_links, '--no-index', '--uptodate',
         expect_stderr=True,
     )
+    assert WARN_NOCOL in result.stderr, str(result)
+    assert 'simple (1.0)' not in result.stdout  # 3.0 is latest
```

Comment:
... yes, looks like what I said above is the case - there's no reason to get the ```--no-columns``` warning here, as that option isn't specified.